dts/bindings/display/sitronix,st7789h2-gpio.yaml

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) 2021, Jeremy LOCHE
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  ST7789H2 320x240 display controller
+
+  Using the GPIOs as bitbanging parallel bus interface for the
+  read/write operations to the display.
+
+compatible: "sitronix,st7789h2-gpio"
+
+include: base.yaml
```

Comment:
please also inherit from `display-controller` and re-use properties

Reply:
Done